inst/htmlwidgets/datatables.js

```diff
@@ -520,11 +520,14 @@ HTMLWidgets.widget({
               filter.val(v);
             }
           });
-          var formatDate = function(d) {
+          var formatDate = function(d, isoFmt = false) {
```

Comment:
Default parameter values are not supported in IE: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters. The conservative approach is to test if `isoFmt` is `undefined`.

Reply:
Thanks for sharing this.